File: src/main/java/io/jenkins/plugins/gamekins/util/Pair.java
package io.jenkins.plugins.gamekins.util;

public class Pair<T, E> {

    private final T first;
    private final E second;

    public Pair(T first, E second) {
        this.first = first;
        this.second = second;
    }

    public T getFirst() {
        return first;
    }

    public E getSecond() {
        return second;
    }
}

File: src/main/kotlin/org/gamekins/util/Pair.kt
package org.gamekins.util

import java.io.Serializable

data class Pair<out A, out B>(
    val first: A,
    val second: B
) : Serializable {

    /**
     * Returns string representation of the [Pair] including its [first] and [second] values.
     */
    override fun toString(): String = "($first, $second)"
}